### PR TITLE
Reject the promise if the video fails to load in imagebitmap/common.js.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22479)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/326348512)

**Comment by Ms2ger** [comment]
@youennf fyi; this change should help this test fail instead of timing out on macOS

File: 2dcontext/imagebitmap/common.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ function makeVideo() {
         video.oncanplaythrough = function() {
             resolve(video);
         };
+        video.onerror = reject;
```
Comment (youennf) [inline 2dcontext/imagebitmap/common.js]: Looks good to add onerror in any case.
Test might fail in WebKit as the test tries to load ogv.
Can we use common/media.js to abstract this test of specific video formats?

**Review by youennf — CHANGES_REQUESTED** [review]

File: 2dcontext/imagebitmap/common.js (hunk 1 above)
Comment (youennf) [reply 2dcontext/imagebitmap/common.js]: Look for getVideoURI for examples

Comment (Ms2ger) [reply 2dcontext/imagebitmap/common.js]: I think the needed pattern.mp4 doesn't exist yet; someone would need to create it first.

**Review by youennf — APPROVED** [review]
OK, let's go with this rejection for now.
Having pattern.mp4 would be nice.

**Pull request merged**